packages/monitoring/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@groww-tech/monitoring",
+  "version": "0.0.1",
+  "description": "Web Monitoring is a service used to inject prometheus monitoring configuration",
+  "main": "src/index.ts",
```

Comment:
serve it from dist folder and add module and types too. Take reference from ella.